modules/installation-osp-default-kuryr-deployment.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-installer-kuryr.adoc
+
+[id="installation-osp-default-kuryr-deployment_{context}"]
+= Resource guidelines for installing {product-title} on OpenStack with Kuryr
+
+When using Kuryr SDN, the Pods, Services, namespaces, and network policies are
+using resources from the {rh-openstack} quota; this increases the minimum
+requirements. Kuryr also has some additional requirements on top of what a
+default install requires.
+
+Use the following quota to satisfy a default cluster's minimum requirements:
+
+.Recommended resources for a default {product-title} cluster on {rh-openstack} with Kuryr
+
+[options="header"]
+|================================
+|Resource              | Value
+|Floating IP addresses | 3 - plus the expected number of Services of LoadBalancer type
+|Ports                 | 1500 - 1 needed per Pod
+|Routers               | 1
+|Subnets               | 250 - 1 needed per Namespace/Project
+|Networks              | 250 - 1 needed per Namespace/Project
+|RAM                   | 112 GB
+|vCPUs                 | 28
+|Volume storage        | 175 GB
+|Instances             | 7
+|Security groups       | 250 - 1 needed per Service and per NetworkPolicy
+|Security group rules  | 1000
+|Swift containers      | 2
+|Swift objects         | 1
+|Swift available space | 10 MB or more
+|================================
+
+[NOTE]
+====
+The number of ports required is actually larger than the number of Pods. Kuryr
+uses ports pools to have pre-created ports ready to be used by Pods and speed up
+the Pods booting time.
+====
+
+[NOTE]
```

Comment:
Just flagging as an area for possible future improvement - this is a lot of notes in a row.

Reply:
I was struggling with that as well :skull:. Do you have any suggestions for how to break them up? I suppose we could break them out of the **NOTE** sidebars and present them as normal text. I agree this is note overload.

Reply:
OK; I went ahead and pulled those notes into regular text to avoid notes galore. I think it's more visually appealing now. Thanks for pointing it out!